### DISCUSSION TITLE
EP2220: How to i get the display off?

### DISCUSSION THREAD

**Discussion by ch-rit** [TillFleisch/ESPHome-Philips-Smart-Coffee#89] [General]
On this picture, it looks like the display is clipped out. I couldnt manage to get my display out and was scared i'd break it.
I found a ifixit-Post, where the entire thing is disassembled, just to get to the display. 

![Alt text](https://github.com/TillFleisch/ESPHome-Philips-Smart-Coffee/raw/main/images/machine_inside.jpg)

**Comment by TillFleisch** [comment]
For anyone having the same question/problem:
You can clip out the display unit from the coffee machine with the help of a spudger tool (or a knive if you're brave enough).
Insert the tool between the chrome colored part of the display and the coffee machine body and gently pry them open.
Once the first clip is loose, the remaining ones come loose more easily.
